app-backend/db/src/main/scala/com/azavea/rf/database/Dao.scala

```diff
@@ -147,8 +148,11 @@ object Dao {
         .run
     }
 
-    def exists(id: UUID): ConnectionIO[Boolean] = {
-      list(0, 1).map(!_.isEmpty)
+    def exists: ConnectionIO[Boolean] = {
```

Comment:
This is back in a new way because the previous way compiled but didn't actually use the id passed to it...

Reply:
Lol ok cool